src/cmd/cmd_sym.ml

```diff
@@ -39,6 +40,7 @@ type parameters =
 let run_file ~parameters ~source_file =
   let { unsafe; rac; srac; entry_point; invoke_with_symbols; _ } = parameters in
   let* m = Compile.File.until_validate ~unsafe ~rac ~srac source_file in
+  Cmd_call_graph.compute_distances m entry_point;
```

Comment:
We should do this *only* if the `Smart` strategy was given, otherwise it'll makes other strategies slow for no good reason.